### PR TITLE
Warn against navx reset in non disabled method

### PR DESCRIPTION
**Pull request by twinters007**
Also fixed a docs issue

### CODE REVIEW

File: robotpy_ext/common_drivers/navx/ahrs.py [hunk 1]
```diff
@@ -339,8 +341,7 @@ def getActualUpdateRate(self):
         an actual rate of 66Hz (200 / (200 / 58), using integer
         math.
         
-        :returns: Returns the current actual update rate in Hz
-        (cycles per second).
+        :returns: Returns the current actual update rate in Hz (cycles per second).
```
Comment (Twinters007) [inline robotpy_ext/common_drivers/navx/ahrs.py]: This fixes a docs issue

**Comment by auscompgeek** [comment]
I don't see where in any of the navX code you'd have a 5 second delay?

**Comment by twinters007** [comment]
Hmm it might just be in init when there's a delay...

File: robotpy_ext/common_drivers/navx/ahrs.py [hunk 2]
```diff
@@ -275,6 +275,8 @@ def zeroYaw(self):
         subtracted from subsequent yaw values reported by
         the getYaw() method.
         """
+        if 'disabled' not in inspect.stack()[1][3]:
```
Comment (virtuald) [inline robotpy_ext/common_drivers/navx/ahrs.py]: This is not a good idea. Instead, ask the DS whether the robot is in disabled mode?

Comment (twinters007) [reply robotpy_ext/common_drivers/navx/ahrs.py]: ooh I like that

**Pull request closed without merging**